mne/inverse_sparse/mxne_optim.py

```diff
@@ -110,7 +110,7 @@ def prox_l1(Y, alpha, n_orient):
     [ True  True False False]
     """
     n_positions = Y.shape[0] // n_orient
-    norms = np.sqrt(np.sum((np.abs(Y) ** 2).T.reshape(-1, n_orient), axis=1))
+    norms = np.sqrt((Y * Y.conj()).real.T.reshape(-1, n_orient).sum(axis=1))
```

Comment:
... and actually, can't we have complex data types in raw, and thus complex data in epochs, and thus complex data in evoked, thus complex data at this stage...?